src/crimson/os/seastore/cache.h

```diff
@@ -1984,4 +1976,20 @@ class Cache {
 };
 using CacheRef = std::unique_ptr<Cache>;
 
+class CacheProxyImpl final : public CacheProxy {
```

Comment:
Seems it is indentical (and simpler) to make `Cache` to inherit from `CacheProxy` ?